rust/xaynet-mobile/src/ffi/participant.rs

```diff
@@ -120,6 +133,8 @@ pub unsafe extern "C" fn xaynet_ffi_participant_new(settings: *const Settings) -
 ///     model once the [`PARTICIPANT_SHOULD_SET_MODEL`] flag is set.
 ///   - [`PARTICIPANT_SHOULD_SET_MODEL`]: if set, then the participant should set its
 ///     model, by calling [`xaynet_ffi_participant_set_model()`]
+///   - [`PARTICIPANT_NEW_GLOBAL_MODEL`]: if set, the participant can fetch the new global
+///     model, by calling [`xaynet_ffi_participant_global_model()`]
```

Comment:
this is fine, but noticed in line 109/110 can remove "tries" / "attempts" from the doc string. also maybe an unintended line break in line 145.

Reply:
> this is fine, but noticed in line 109/110 can remove "tries" / "attempts" from the doc string.
sorry I don't understand what you mean here

Reply:
```rust
/// Drive the participant internal state machine. Every tick, the state machine tries
/// attempts to perform a small work unit.
```
can either chop `tries` or `attempts`, that's all :)

Reply:
Ahh I see, I was looking at the wrong line. Thanks 👍

Reply:
fixed